randomizer/CompileHints.py

```diff
@@ -797,6 +799,9 @@ def compileHints(spoiler: Spoiler) -> bool:
                 hint_distribution[HintType.Multipath] = len(multipath_dict_hints.keys())
                 maxed_hint_types.append(HintType.Multipath)
             valid_types.append(HintType.Multipath)
+        # If somehow you threaded the needle with no valid hint types, you'll get joke hints whether you like it or not
+        if len(valid_types) == 0:
+            valid_types = []
```

Comment:
Is this supposed to be  valid_types = [HintType.Joke]? This looks like it does nothing.

Reply:
you get today's gold star